counterparty-lib/counterpartylib/lib/restapi.py

```diff
@@ -0,0 +1,121 @@
+import argparse
```

Comment:
I think we should call this file just `api.py` and move the existing `api.py` to `v1/api.py`?

Reply:
yep! sounds good!